backend/src/Designer/Services/Implementation/ProcessModeling/ProcessModelingService.cs

```diff
@@ -0,0 +1,68 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+using Altinn.Studio.Designer.Infrastructure.GitRepository;
+using Altinn.Studio.Designer.Models;
+using Altinn.Studio.Designer.Services.Interfaces;
+
+namespace Altinn.Studio.Designer.Services.Implementation.ProcessModeling
+{
+    public class ProcessModelingService : IProcessModelingService
+    {
+        private readonly IAltinnGitRepositoryFactory _altinnGitRepositoryFactory;
+        public ProcessModelingService(IAltinnGitRepositoryFactory altinnGitRepositoryFactory)
+        {
+            _altinnGitRepositoryFactory = altinnGitRepositoryFactory;
+        }
+
+        private string TemplatesFolderIdentifier(Version version) => string.Join(".", nameof(Services), nameof(Implementation), nameof(ProcessModeling), "Templates", $"v{version.Major}");
```

Comment:
Might be that I just dont understand this code, but does it take into account if it receives an unknown version?